pkg/deploy/broker.go

```diff
@@ -82,6 +88,15 @@ func Broker(options *BrokerOptions, clientProducer client.Producer, status repor
 		return status.Error(err, "error creating globalCIDR configmap on Broker")
 	}
 
+	if err = clustersetip.ValidateExistingClustersetIPNetworks(ctx, clientProducer.ForGeneral(), options.BrokerNamespace); err != nil {
+		return status.Error(err, "error validating existing clustersetIPCIDR configmap")
+	}
+
+	if err = clustersetip.CreateConfigMap(ctx, clientProducer.ForGeneral(), options.BrokerSpec.ClustersetIPEnabled,
+		options.BrokerSpec.ClustersetIPCIDRRange, 0, options.BrokerNamespace); err != nil {
+		return status.Error(err, "error creating clustersetIPCIDR configmap on Broker")
+	}
```

Comment:
Shouldn't the [broker controller](https://github.com/submariner-io/submariner-operator/blob/268a9960c6b70ddbf6fd751fe6eae6ebb24f4077/controllers/submariner/broker_controller.go#L85) handle this as it does for globalnet?

Reply:
> Shouldn't the [broker controller](https://github.com/submariner-io/submariner-operator/blob/268a9960c6b70ddbf6fd751fe6eae6ebb24f4077/controllers/submariner/broker_controller.go#L85) handle this as it does for globalnet?

Yeah, it should but that is for addon use case where addon operator runs broker controller. There is no broker controller when usin subctl to deploy broker, that is why we do this for globlanet too in subctl.

https://github.com/submariner-io/subctl/blob/e6b9fe5154cca34f9fe29b7dc7ee679bfbd8da23/pkg/deploy/broker.go#L80

Reply:
Actually, the broker controller always runs in the operator pod - it's activated by creation of the `Broker` resource.

The behavior wrt the addon vs subctl is actually the opposite: the addon creates the globalnet `ConfigMap` and does not create the `Broker` resource while subctl does both tho I'm not sure why.  It's fine if we do the same for clusterset IP here to mirror globalnet behavior but perhaps we should also do the same in the broker controller otherwise it renders the new fields in the Broker CRD meaningless. We could revisit  the duplicate behavior later. 

Also note that the helm charts does neither but it's moot b/c we expect the user to pass in the global CIDR when deploying the operator.